src/idm/user.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+import parseUrl from 'url-parse';
+
+const idm = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @summary Get the user information
```

Comment:
one gives you the info for the user that is currently logged in. I updated the names and the documentation